images/ipfailover/keepalived/lib/utils.sh

```diff
@@ -36,26 +188,26 @@ function scrub() {
 function expand_ip_ranges() {
   local vips=${1:-""}
   local expandedset=()
+  local iprange
+  local newip
 
-  for iprange in $(echo "$vips" | sed 's/[^0-9\.\,-]//g' | tr "," " "); do
```

Comment:
Not sure why it was that way before, as the sed s command uses single quotes (so it's basically a literal and needs no escaping, save for single quotes), and ',' isn't a sed regexp special character. '\.' on the other hand does need the quote for the regexp to be meaningful, so I'm guessing it was more of a "it won't hurt if we add it" kind of thing... If someone wants to shed some light on it, by all means do!